docs/usage.rst

```diff
@@ -2,8 +2,353 @@
+``test/genomes`` directory (see section `Creation of Phenotrex Input Features`_):
+
+.. code-block:: console
+
+    $ phenotrex predict \
+        --genotype test/T3SS.test.genotype \
+        --classifier T3SS.pkl \
+        --min_proba 0.6 \
+        --verb > T3SS.test_predictions.tsv
+
+
+The ``predict`` command outputs prediction results directly to stdout. When redirecting stdout to
+a file, this results in a 3-column TSV file of the following shape:
+
+..  code-block::
+
+    # Trait: T3SS
+    Identifier	Trait present	Confidence
+    GCF_000006645.1.fna.gz	YES	0.8604
+    GCF_000006665.1.fna.gz	YES	0.8675
+    GCF_000006825.1.fna.gz	NO	0.6617
+    GCF_000007165.1.fna.gz	YES	0.6771
+    GCF_000007205.1.fna.gz	YES	0.8261
+    GCF_000007445.1.fna.gz	YES	0.8183
+    ...
+
+
+Lines starting with ``#`` represent metadata, in this case the trait name saved in the used model artifact.
+
+
+Explanation of Phenotrex Predictions
+------------------------------------
+In addition to providing predicted phenotype labels and confidence measures,
+phenotrex can provide additional explanations of its decision process.
+This can help debug faulty hyperparameter configurations and help identify errors in the training data.
+Model explanation is done by gauging the importance of input features identified in genomes at training and prediction time.
+
+Feature Importance at Training Time
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+The relative impact of features learned by phenotrex models is output at training time when the flag
+``--weights`` is added to the ``phenotrex train {xgb,svm}`` command. The meaning of the importance
+differs depending on the selected ML algorithm: when using XGB, the measure represents the overall
+importance of that feature in the decision process of the model (irrespective of the final prediction),
+when using SVM, the measure correlates with the probability of calling YES (positive
+values) or NO (negative values) for the phenotype in question.
+
+Feature Importance at Prediction Time
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+A second, and arguably more useful type of explanation can be computed at prediction time.
+For each predicted genome, a list of features is created which, either by presence or absence,
+contributed most to the prediction output for that genome.
+Feature importance is represented by `SHAP`_ (SHapley Additive exPlanations) values.
+The sum of SHAP values of all features considered by the model is directly related to the probability
+of calling YES for the phenotype and genome in question.
+
+.. note::
+
+    Feature explanation at prediction time is implemented by the `shap package`_, which efficiently
+    computes the required explanations for XGB models with de facto zero overhead.
+    For SVM models however, this calculation can be extremely costly. We thus suggest that for
+    use cases where model introspectability is important, XGB should be preferred over SVM.
```

Comment:
Is that a word? My personal preference would be explainability or interpretability. Also, we are talking about explanations all the time.

Reply:
fair